src/test/components/__snapshots__/ThreadActivityGraph.test.js.snap

```diff
@@ -1182,6 +1182,226 @@ Array [
     "set fillStyle",
     "#6200a4",
   ],
+  Array [
```

Comment:
This diff is huge, and I admit I haven't looked closely.

I _think_ this is because the order of rendering changed because of the new requestAnimationFrame: previously I believe the ActivityGraph was always happening first, but now it happens more or less in the same order of the page.

(but I'm not sure of this and this is very difficult to assert for such big changes)

Reply:
Yeah, it's not easy to look at these snapshots in general, but at least it shows the errors like null or NaN values. I tried to look at the changes as much as I could and I don't see anything unusal.